app/assets/stylesheets/custom.css.scss

```diff
@@ -420,4 +415,10 @@ form label {
 	top: 0;
 	z-index: 1;
 	left: 0;
+}
+
+
+.about_class {
```

Comment:
should this be padding? or does margin also work?

Reply:
margin is fine here

Reply:
I think both work!